src/test/crimson/test_messenger.cc

```diff
@@ -34,6 +35,29 @@ seastar::logger& logger() {
   return crimson::get_logger(ceph_subsys_ms);
 }
 
+class AbortTimer {
```

Comment:
probably, it'd be simpler to use `seastar::with_timeout()`? i created #38700 to explain the idea.